frontend/app/dashboard/src/views/dashboard/settings/UitpasSettingsView.vue

```diff
@@ -111,25 +166,56 @@ const openSearchOrganizer = async () => {
                 uitpasOrganizerName.value = organizer.name;
                 uitpasOrganizerId.value = organizer.id;
                 await organizationManager.value.patch(patch.value);
-                await pop({ force: true });
+                if (props.onFixedAndEventSelected && uitpasClientCredentialsStatus.value === UitpasClientCredentialsStatus.Ok && uitpasEvent.value) {
+                    await props.onFixedAndEventSelected(uitpasEvent.value, navigationActions);
+                }
+                else {
+                    await pop({ force: true });
+                }
             },
-        }),
+        }).setDisplayStyle('popup'),
     );
 };
 
 const openSetUitpasClientCredentialsView = async () => {
-    const u = new UitpasClientIdAndSecret();
-    u.clientId = '';
-    u.clientSecret = '';
-    // TO DO proper getting using endpoint
     present({
         components: [
-            new ComponentWithProperties(SetUitpasClientCredentialsView, {}),
+            new ComponentWithProperties(SetUitpasClientCredentialsView, {
+                onFixed: (navigationActions: NavigationActions) => {
+                    if (props.onFixedAndEventSelected && uitpasEvent.value) {
+                        return props.onFixedAndEventSelected(uitpasEvent.value, navigationActions);
```

Comment:
Hier heb je een probleempje.

SetUitpasClientCredentialsView wordt getoond als 'sheet', en je geeft de navigation actions van die modal mee aan je `onFixedAndEventSelected`. Dat wil zeggen dat je de ene keer een modal zonder navigation controller meegeeft, en de andere keer met. Als je daarna dus je show met replace gaat uitvoeren, gaat het in dit scenario niet werken (en zelfs met navigation controller zou het dan in de sheet getoond worden en niet op de juiste plaats)